### PR TITLE
New api integration

### PR DESCRIPTION
**Pull request by WLeandroDiPaola**
## Summary

added a new service that connects to geek-jokes

## Trello Card

[Card](https://trello.com/c/JVWTOB8m/32-servicios-promises-witter)

### CODE REVIEW

File: app/services/geek-jokes.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+const axios = require('axios');
+
+const getJoke = () => {
+  axios
```
Comment (carzoglioemanuel) [inline app/services/geek-jokes.js]: You are not returning the promise. You will have a problem with the asyncronism.

File: app/services/geek-jokes.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    .get('https://geek-jokes.sameerkumar.website/api?format=json')
```
Comment (carzoglioemanuel) [inline app/services/geek-jokes.js]: Use a environment variable for this URL to avoid the hardcoding

File: app/services/geek-jokes.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    .then(
+      response =>
+        // handle success
```
Comment (carzoglioemanuel) [inline app/services/geek-jokes.js]: remove comment

File: app/services/geek-jokes.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+        response.data.joke
+    )
+    .catch(
+      error =>
+        // handle error
```
Comment (carzoglioemanuel) [inline app/services/geek-jokes.js]: remove comment

File: app/services/geek-jokes.js [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+        error
```
Comment (carzoglioemanuel) [inline app/services/geek-jokes.js]: Return a specific error here.
Also, you can use informative logs to the app registry

File: app/services/geek-jokes.js [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+    )
+    .then(() => {
```
Comment (carzoglioemanuel) [inline app/services/geek-jokes.js]: remove this `.then`

File: app/services/geek-jokes.js [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+      // always executed
```
Comment (carzoglioemanuel) [inline app/services/geek-jokes.js]: remove comment

File: app/routes.js [hunk 8]
```diff
@@ -0,0 +1,11 @@
+// const controller = require('./controllers/controller');
```
Comment (molinajulian) [inline app/routes.js:1]: Delete all comments

File: app/routes.js [hunk 9]
```diff
@@ -0,0 +1,11 @@
...
+const { healthCheck } = require('./controllers/healthCheck');
+const users = require('./controllers/users');
```
Comment (molinajulian) [inline app/routes.js:3]: Could you destruct the object,please?

File: app/services/geek_jokes.js [hunk 10]
```diff
@@ -0,0 +1,17 @@
+const axios = require('axios');
+const { inspect } = require('util');
+
+const logger = require('../logger/index');
+const { apiJokesError } = require('../errors');
+const config = require('../../config').common;
+
+const getJoke = () =>
```
Comment (molinajulian) [inline app/services/geek_jokes.js:8]: You can export directly the function, for example `exports.getJoke`

Comment (molinajulian) [reply app/services/geek_jokes.js:8]: Maybe the name `getRandomJoke`it's more descriptive right? 🤔

File: config/index.js [hunk 11]
```diff
@@ -0,0 +1,55 @@
...
+      header_name: 'authorization',
+      secret: process.env.NODE_API_SESSION_SECRET
+    },
+    headers: {
+      apiDate: process.env.API_DATE || 'X-API-Date',
+      packageVersion: process.env.PACKAGE_VERSION || 'X-Package-Version',
+      nodeVersion: process.env.NODE_VERSION || 'X-Node-Version'
+    },
+    geekJokes: {
+      apiUrl: process.env.GEEK_JOKES_API
```
Comment (molinajulian) [inline config/index.js:49]: Good job 💪

**Review by molinajulian — APPROVED** [review]

**Pull request merged**